packages/sdk-react/src/core/hooks/useFormPersist.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import { SetFieldValue } from 'react-hook-form';
+
+// Persists form state between page reloads
+
+export const useFormPersist = (
+  name: string,
+  getValues: () => { [key: string]: any },
```

Comment:
This type can be more specific; if possible, replace `any` with more defined types of your form values.